mtp_api/apps/core/dashboards.py

```diff
@@ -0,0 +1,224 @@
+import datetime
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.db import models
+from django.forms import MediaDefiningClass
+from django.utils.timezone import now
+from django.utils.translation import gettext_lazy as _
+
+from account.models import Balance
+from transaction.constants import TRANSACTION_CATEGORY, TRANSACTION_SOURCE
+from transaction.models import Transaction
+from transaction.utils import format_amount, format_number
+
+
+class DashboardModule(metaclass=MediaDefiningClass):
+    template = 'core/dashboard/module.html'
+    html_classes = 'module'
+    title = _('Dashboard')
+    enabled = True
+
+
+class TransactionReport(DashboardModule):
+    template = 'core/dashboard/transaction-report.html'
+    title = _('Transaction report')
+    column_count = 3
+
+    class Media:
+        css = {
+            'all': ('core/css/transaction-report.css',)
+        }
+
+    def __init__(self, received_at=None):
+        if not received_at:
+            self.title = _('Today’s transaction report')
+            received_at = now().date()
```

Comment:
Should the default rather be yesterday, as this is when the most recent transactions will be dated?